static/index.html

```diff
@@ -7,8 +7,9 @@
  <title>Drupalcamp Cebu 2018</title>
 
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
- <link rel="stylesheet" href="/assets/style.css">
- <link href="https://fonts.googleapis.com/css?family=Muli:200,400|Saira:400,700" rel="stylesheet">
+ <link rel="stylesheet" href="./css/style.css">
+<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
```

Comment:
I believe google fonts can be added as one line, not two. One line = One request. Faster to download once, than two times.

Reply:
Use this instead:
<link href="https://fonts.googleapis.com/css?family=Merriweather:700|Open+Sans" rel="stylesheet">